build.sbt

```diff
@@ -607,4 +607,4 @@ ThisBuild / publishMavenStyle.withRank(KeyRanks.Invisible) := true
 ThisBuild / sonatypeCredentialHost := sonatypeCentralHost
 Test / publishArtifact := false
 ThisBuild / publishTo := sonatypePublishToBundle.value
-ThisBuild / pomExtra := MavenPublishing.pomNodeSeq()
+ThisBuild / pomExtra := MavenPublishing.pomNodeSeq()
```

Comment:
This change seems unrelated to the PR (and also, please keep terminating new lines also in other files)